documentation/src/main/asciidoc/stories/assembly_cloud_discovery.adoc

```diff
@@ -0,0 +1,30 @@
+[id='cloud_discovery-{context}']
+= Cloud Discovery Mechanisms
+Running {brandname} on hosted services means using certain discovery mechanisms
+to adapt to different network settings between cloud providers. For instance,
```

Comment:
How about "Running {brandname} on hosted services requires using discovery mechanisms that are adapted to the network constraints imposed by the individual cloud providers" ?